stm32-modules/thermocycler-refresh/firmware/system/system_hardware.c

```diff
@@ -0,0 +1,68 @@
+#include "stm32g4xx_hal.h"
+#include "stm32g4xx_hal_rcc.h"
+#include "stm32g4xx_hal_cortex.h"
+#include "system_hardware.h"
+
+void system_hardware_setup(void) {
+    GPIO_InitTypeDef gpio_init = {
+      .Pin = GPIO_PIN_6,
+      .Mode = GPIO_MODE_OUTPUT_PP,
+      .Pull = GPIO_NOPULL,
+      .Speed = GPIO_SPEED_FREQ_LOW
+    };
+    __HAL_RCC_GPIOE_CLK_ENABLE();
+    HAL_GPIO_Init(GPIOE, &gpio_init);
+}
+
+// This is the start of the sys memory region from the datasheet. It should be the
```

Comment:
are these values right? should make sure and change the comment pr update and remove

Reply:
Good catch! Updating with values for the G491